### DISCUSSION TITLE
Not able to deploy my sveltekit website on github pages.

### DISCUSSION THREAD

**Discussion by arujjval** [sveltejs/kit#12085] [Q&A]
### Describe the bug

I have made a website using sveltekit along with tailwindCSS. Since its a static website, so I want to deploy it on github pages. but i am facing issue.
I refer this page https://kit.svelte.dev/docs/adapter-static
and  followed all the instructions. I also read some blogs but nothing seems to work. Help me resolve this issue.

this is the repo https://github.com/arujjval/computer_networking
and it is only rendering readme.md (even after i have put .nojekyll file in static folder)

### Reproduction

this is the repo https://github.com/arujjval/computer_networking

### System Info

```Shell
System:
    OS: Windows 11 10.0.22631
    CPU: (12) x64 AMD Ryzen 5 6600H with Radeon Graphics
    Memory: 5.88 GB / 15.19 GB
  Binaries:
    Node: 20.11.0 - C:\Program Files\nodejs\node.EXE
    npm: 10.3.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Chromium (123.0.2420.65)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @sveltejs/adapter-auto: ^3.0.0 => 3.1.1
    @sveltejs/adapter-cloudflare: ^4.2.0 => 4.2.0
    @sveltejs/adapter-static: ^3.0.1 => 3.0.1
    @sveltejs/kit: ^2.5.5 => 2.5.5
    @sveltejs/vite-plugin-svelte: ^3.0.0 => 3.0.2
    svelte: ^4.2.7 => 4.2.12
    vite: ^5.0.3 => 5.2.6
```

### Severity

annoyance

**Comment by brunnerh** [comment]
Something is off, where is the `gh-pages` or other deployment branch?

It looks like you just pointed GH pages to the main branch rather than a separate one containing the *built* project.

_Edit:_ Apparently there are ways to deploy without having an explicit extra branch, content still looks like it's pointing to main though.

**Comment by teemingc** [comment]
I've converted your issue into a discussion since issues should really only report issues with the framework or feature requests. Discussions are more suitable for getting help with debugging your project.

I've prepared an example GitHub pages repository with SvelteKit if that helps your case https://github.com/eltigerchino/eltigerchino.github.io
